pkg/operator/loglevel/logging_controller.go

```diff
@@ -15,14 +15,23 @@ type LogLevelController struct {
 	// for unit tests only
 	setLogLevelFn func(operatorv1.LogLevel) error
 	getLogLevelFn func() (operatorv1.LogLevel, bool)
+
+	defaultLogLevel operatorv1.LogLevel
 }
 
-// sets the klog level based on desired state
+// NewClusterOperatorLoggingController sets a klog level for the operator based on the operator config.
+// If the loglevel is not set the default "Normal" level will be used.
 func NewClusterOperatorLoggingController(operatorClient operatorv1helpers.OperatorClient, recorder events.Recorder) factory.Controller {
+	return NewClusterOperatorLoggingControllerWithLogLevel(operatorClient, operatorv1.Normal, recorder)
+}
+
+// NewClusterOperatorLoggingControllerWithLogLevel sets a klog level for the operator based on the operator config.
```

Comment:
the comment does not match

Reply:
what does not match?

Reply:
it will still set what you specify in operator CR :-) but if you don't it will default

Reply:
NewClusterOperatorLoggingControllerWithLogLevel sets a klog level for the operator based on the operator config, using the given default log level if the operator config does not specify anything.